index-handlers/src/main/java/no/sikt/nva/nvi/index/model/NviCandidateMessageBody.java

```diff
@@ -6,7 +6,9 @@
 import com.fasterxml.jackson.databind.annotation.JsonSerialize;
 import java.net.URI;
 import java.util.List;
+import nva.commons.core.JacocoGenerated;
 
+@JacocoGenerated
 @JsonAutoDetect(fieldVisibility = Visibility.ANY)
 @JsonSerialize
 public record NviCandidateMessageBody(@JsonProperty(PUBLICATION_BUCKET_URI) String publicationBucketUri,
```

Comment:
I think this class can be deleted now?

Reply:
It cant, this object is never tested in commons

Reply:
It is still used in indexHandler, cause dynamoDbevent does not contain all the data we need to index